samples/winapi_drivelist/main.c

```diff
@@ -56,11 +57,25 @@ int main(void)
         debugPrint("%s\n", drive);
         while(*drive++);
     }
-    debugPrint("\ndone");
 
-    while(1) {
-        Sleep(2000);
+    debugPrint("\nDone!");
+
+unmount_e:
+    ret = nxUnmountDrive('E');
+    if (!ret) {
+        error = GetLastError();
+        debugPrint("\nFailed to unmount E: drive! Error code: %x\n", error);
+    }
+
+unmount_c:
+    ret = nxUnmountDrive('C');
+    if (!ret) {
+        error = GetLastError();
+        debugPrint("\nFailed to unmount C: drive! Error code: %x\n", error);
     }
 
-    return 0;
+end:
```

Comment:
nit: `end` is fine, but I'd suggest `waitIndefinitely` probably: it's most descriptive

Reply:
Will name it `sleepForever`, should be good enough.